scripts/firedrake-install

```diff
@@ -1313,6 +1352,10 @@ with environment(**compiler_env):
     with pipargs("--no-deps"):
         if options["slepc"]:
             build_and_install_slepc()
+
+        if options["opencascade"]:
+            build_and_install_pythonocc()
+
```

Comment:
pythonocc explicitly guarantees (https://github.com/tpaviot/pythonocc-core) that any version of pythonocc in the 0.18.x range (since 2017) will compile against any oce in the 0.18.x range. However, I agree it's probably a good idea to pin to a specific version, once I figure out how ...

Reply:
Version now pinned.